.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "src/fl_classes"]
 	path = src/fl_classes
-	url = git@github.com:MiguelGuthridge/fl_classes.git
```

Comment:
Is there any particular reason why this should be changed?

Reply:
I'm not sure why that changed to be honest, I had some troubles getting the submodule to fetch on my end and I guess it changed it to the URL while I was messing around with it. Shouldn't be much of a difference but I can revert it if you'd like?

Reply:
The main reason it was SSH rather than HTTPS was since then pushing to it (which I almost always do when working on this repo) doesn't require a username and password each time. I'll investigate to see if there's some way I can get this working so that it just picks whichever protocol works properly rather than requiring one or the other.